### ISSUE TITLE
Fails to compile if voice feature not present

### ISSUE THREAD

**Issue by tahahawa** [zeyla/serenity#73]
If the voice feature is not enabled, it fails to compile with the message:

```
error[E0463]: can't find crate for `byteorder`
   --> ~/.cargo/git/checkouts/serenity-3ee4e93465cd5f21/6a1c257/src/lib.rs:102:1
    |
102 | extern crate byteorder;
    | ^^^^^^^^^^^^^^^^^^^^^^^ can't find crate
```

Workaround for now: enable voice feature.

**Comment by zeyla** [comment]
I pre-maturely published a v0.2.0 commit bump, which I ended up not publishing a release for due to a few issues. A commit (a1c035d) resolves this issue which has just been pushed. A `cargo update` should resolve this.

**Comment by tahahawa** [comment]
Thanks! I'll close this issue now. 

Also, thanks in general for making the lib
